### ISSUE TITLE
start-notebook.sh no longer works?

### ISSUE THREAD

**Issue by cboettig** [data-8/jupyterhub-deploy#34]
Hi @ryanlovett , 

did something change in a recent re-build of the docker image?  Previously running the docker image (discussed in #2) with something like:

```
docker run --rm -p 8888:8888 cboettig/ds8 start-notebook.sh

```

worked just fine but pulling a recent image and running now results in the error:

```
id: jovyan: no such user
/usr/local/bin/start-notebook.sh: line 6: [: 1000: unary operator expected
No passwd entry for user 'jovyan'
```

looks like maybe you've removed the default user inherited from `minimal-notebook`?

**Comment by ryanlovett** [comment]
We altered the CMD but it shouldn't affect your start-notebook.sh bootstrap. Unfortunately the scipy-notetebook, systemuser, and singleuser containers in our docker stack are not tagged so if/when they break in master, it breaks us. The cboettig/ds8 container is currently using a bootstrap CMD that's more appropriate for a Hub environment, so it might simplify things to switch your FROM to [scipy-notebook](https://hub.docker.com/r/jupyter/scipy-notebook/tags/) and then choosing a specific commit tag.

I suppose ideally we could have a ds8 and ds8+systemuser pair akin to https://github.com/jupyter/docker-stacks/pull/162.

**Comment by cboettig** [comment]
hm, right, not sure at what level the `jovyan` user disappeared.  I could drop back to a (tagged) sci-py notebook (or further); but this rather defeats the idea of running the exact environment you have on the data8.berkeley.edu and ds8.berkeley.edu servers.  (Or it would if you put your image on the hub instead of building it locally!  The Dockerfile in cboettig/ds8 is just a copy of the Dockerfile you have in this repo; ideally I wouldn't be maintaining a Dockerfile at all).

Sure I can always write my own Dockerfile on top of some other part of the Jupyter stack; that's what I was doing until recently (building on jupyter/datascience image and installing `datascience` package), but I was hoping to avoid that and get a close as possible to what was on the servers; particularly since frequent changes to `Tables` break my lessons or my students homeworks in between writing them and grading or presenting them and I'd like to catch those changes at home first....

**Comment by ryanlovett** [comment]
I agree with everything you just said. :) I think John is the owner of the data-8 repo so I'll need to contact him about pairing it up with a corresponding docker hub account.

**Comment by cboettig** [comment]
cool beans, thanks.

**Comment by SamLau95** [comment]
@ryanlovett I'm also an owner so I think I can also help get things set up if need be.
